From 9734c1a065a5217687c5a485a24b149664c46cfa Mon Sep 17 00:00:00 2001
From: "tong.gao" <tong.gao@amlogic.com>
Date: Wed, 26 Apr 2023 19:41:53 +0800
Subject: [PATCH] dts:yocto-k5.15  [1/1]

PD#SWPL-120168

Problem:
Memory allocation fail

Solution:
add code

Verify:
local

Signed-off-by: tong.gao <tong.gao@amlogic.com>
Change-Id: I6ec1675dcaa7aca7a3c9782d5ccf9f19e9a561ed
---
 arch/arm64/boot/dts/amlogic/t7c_a311d2_an400_linux.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/amlogic/t7c_a311d2_an400_linux.dts b/arch/arm64/boot/dts/amlogic/t7c_a311d2_an400_linux.dts
index 64fc2a94..2b725887 100644
--- a/arch/arm64/boot/dts/amlogic/t7c_a311d2_an400_linux.dts
+++ b/arch/arm64/boot/dts/amlogic/t7c_a311d2_an400_linux.dts
@@ -166,7 +166,8 @@
 		ion_fb_reserved:linux,ion-fb {
 			compatible = "shared-dma-pool";
 			reusable;
-			size = <0x0 0x0>;
+			/* 1920x1080x4x6+3840x2160x4x3 round up 4M align */
+			size = <0x0 0x10000000>;
 			alignment = <0x0 0x400000>;
 			alloc-ranges = <0x0 0x0 0x0 0xe0000000>;
 		};
-- 
2.25.1

